### PR TITLE
Improve contribution docs + clean up AndroidManifest.xml

### PR DESCRIPTION
**Pull request by ianthetechie**
This closes issue #96.

### CODE REVIEW

**Comment by ianthetechie** [comment]
Good call; adding a changelog entry.

I also just realized that the section about adding/updating localization may also be irrelevant now? I can't find MapLibre on Transifex. Should we cut that too?

**Comment by boldtrn** [comment]
> I also just realized that the section about adding/updating localization may also be irrelevant now? I can't find MapLibre on Transifex. Should we cut that too?

Yeah, I don't think we are using Transifex for this project 👍

File: libandroid-navigation/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,19 +4,12 @@
 
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (Fabi755) [inline libandroid-navigation/src/main/AndroidManifest.xml:7]: Well spotted!

File: libandroid-navigation/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -4,19 +4,12 @@
```
[the remark this answers is not in the document]
Comment (ianthetechie) [reply libandroid-navigation/src/main/AndroidManifest.xml:4]: Good catch ;) Removing that now.

**Comment by boldtrn** [comment]
Thanks for the fixes @ianthetechie 👍

**Pull request merged**